var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -113,6 +113,14 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
 
     # https://gcc.gnu.org/install/prerequisites.html
     depends_on('gmp@4.3.2:')
+    # mawk is not sufficient for go support
+    depends_on('gawk@3.1.5:')
```

Comment:
```suggestion
    depends_on('gawk@3.1.5:', when='languages=go')
```
Is this linked to or can it be type build?

Reply:
It can be type build as far as I know, happy to add that. It's listed as a dependency for all configurations, golang happens to be where I noticed the issue, so I'm not sure about limiting it to when the go language is active.

Reply:
```suggestion
    depends_on('gawk@3.1.5:', type='build')
```